docs/contributing.rst

```diff
@@ -54,103 +54,58 @@ where appropriate.
+
+If you have trouble getting the full ``tox`` suite to run locally, it is
+generally sufficient to open a pull request and let Github and Travis run
+integration tests for you.
 
 .. _integration:
 
-Integration testing with the boulder CA
+Integration testing with the Boulder CA
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Generally it is sufficient to open a pull request and let Github and Travis run
-integration tests for you.
-
-However, if you prefer to run tests, you can use Vagrant, using the Vagrantfile
-in Certbot's repository. To execute the tests on a Vagrant box, the only
-command you are required to run is::
-
-  ./tests/boulder-integration.sh
-
-Otherwise, please follow the following instructions.
+To run integration tests locally, you need Docker and docker-compose installed
+and working. Fetch and start Boulder using:
 
-macOS users: Run ``./tests/mac-bootstrap.sh`` instead of
-``boulder-start.sh`` to install dependencies, configure the
-environment, and start boulder.
+  ./tests/boulder-fetch.sh
 
-Otherwise, install `Go`_ 1.5, ``libtool-ltdl``, ``mariadb-server`` and
-``rabbitmq-server`` and then start Boulder_, an ACME CA server.
+Run the integration tests using:
 
-If you can't get packages of Go 1.5 for your Linux system,
-you can execute the following commands to install it:
-
-.. code-block:: shell
-
-  wget https://storage.googleapis.com/golang/go1.5.3.linux-amd64.tar.gz -P /tmp/
-  sudo tar -C /usr/local -xzf /tmp/go1.5.3.linux-amd64.tar.gz
-  if ! grep -Fxq "export GOROOT=/usr/local/go" ~/.profile ; then echo "export GOROOT=/usr/local/go" >> ~/.profile; fi
-  if ! grep -Fxq "export PATH=\\$GOROOT/bin:\\$PATH" ~/.profile ; then echo "export PATH=\\$GOROOT/bin:\\$PATH" >> ~/.profile; fi
-
-These commands download `Go`_ 1.5.3 to ``/tmp/``, extracts to ``/usr/local``,
-and then adds the export lines required to execute ``boulder-start.sh`` to
-``~/.profile`` if they were not previously added
-
-Make sure you execute the following command after `Go`_ finishes installing::
-
-  if ! grep -Fxq "export GOPATH=\\$HOME/go" ~/.profile ; then echo "export GOPATH=\\$HOME/go" >> ~/.profile; fi
-
-Afterwards, you'd be able to start Boulder_ using the following command::
-
-  ./tests/boulder-start.sh
-
-The script will download, compile and run the executable; please be
-patient - it will take some time... Once its ready, you will see
-``Server running, listening on 127.0.0.1:4000...``. Add ``/etc/hosts``
-entries pointing ``le.wtf``, ``le1.wtf``, ``le2.wtf``, ``le3.wtf``
-and ``nginx.wtf`` to 127.0.0.1.  You may now run (in a separate terminal)::
-
-  ./tests/boulder-integration.sh && echo OK || echo FAIL
-
-If you would like to test `certbot_nginx` plugin (highly
-encouraged) make sure to install prerequisites as listed in
-``certbot-nginx/tests/boulder-integration.sh`` and rerun
-the integration tests suite.
-
-.. _Boulder: https://github.com/letsencrypt/boulder
-.. _Go: https://golang.org
+  ./tests/boulder-integration.sh
```

Comment:
This also doesn't seem to be working? I'm getting `ConnectionError: HTTPConnectionPool(host='localhost', port=4000): Max retries exceeded with url: /directory (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7f3f07af4210>: Failed to establish a new connection: [Errno 111] Connection refused',))`